numba/unicode_support.py

```diff
@@ -251,15 +251,31 @@ def _PyUnicode_ToLowercase(ch):
 
 @register_jitable
 def _PyUnicode_ToLowerFull(ch, res):
-    raise NotImplementedError
+    ctype = _PyUnicode_gettyperecord(ch)
```

Comment:
Needs a reference back to the cpython impl in `Objects/unitcodectype.c`

Reply:
Ok, I will add it.